agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -460,13 +462,68 @@ bool slave_thread::handle_cmdu_control_ieee1905_1_message(Socket *sd,
     auto rx_mid = rx_cmdu_message_header->message_id();
 
     switch (cmdu_message_type) {
-    /* For Example:
+    case ieee1905_1::eMessageType::CHANNEL_PREFERENCE_QUERY_MESSAGE: {
+
+        LOG(INFO) << "received CHANNEL_PREFERENCE_QUERY_MESSAGE";
+
+        // build channel preference report
+        auto cmdu_tx_header =
+            cmdu_tx.create(rx_mid, ieee1905_1::eMessageType::CHANNEL_PREFERENCE_REPORT_MESSAGE);
```

Comment:
rebase and use cmdu_rx.getMessageId() directly, don't use another variable.

Reply:
This is the first message of 1905.1 we handle on son_slave, and it is more convenient to have a copy of it which can later be used by every message handler.

Reply:
ok